lib/btrfs_test.pm

```diff
@@ -72,6 +74,17 @@ sub snapper_nodbus_restore {
     die 'DBus service should be inactive, but it is active' if ($ret == 0);
     script_run('systemctl default', timeout => 600, die_on_timeout => 0);
     my $tty = get_root_console_tty;
+
+    if (is_sle('<15-SP3') && !defined(my $match = check_screen("tty$tty-selected", 120))) {
+        record_soft_failure("bsc#1185098 - [Build 15.82] openQA test fails in snapper_undochange - logind fails after return back from rescue");
+        select_console('log-console');
+        if (script_run('systemctl is-active getty@tty2.service') == 3) {
```

Comment:
We should have the best practice of `script_run` together with `die_on_timeout` flag.
https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1010

```suggestion
        if (script_run('systemctl is-active getty@tty2.service', die_on_timeout => 1) == 3) {
```

Reply:
Nice catch, I always forget about this option. Thanks!

Reply:
http://kepler.suse.cz/tests/17407
http://kepler.suse.cz/tests/17408